compiler-core/src/language_server/tests/action.rs

```diff
@@ -79,6 +79,7 @@ const CONVERT_TO_FUNCTION_CALL: &str = "Convert to function call";
 const INLINE_VARIABLE: &str = "Inline variable";
 const CONVERT_TO_PIPE: &str = "Convert to pipe";
 const INTERPOLATE_STRING: &str = "Interpolate string";
+const REPLACE_SPREAD_WITH_IGNORED_FIELDS: &str = "Replace `..` with ignored fields";
```

Comment:
Not totally sure about this name. What else might we call it?

"Fill fields" or "fill unused fields" to match "fill labels"?

Reply:
Fill unused fields sounds great! I'd not use labels because there might be unlabelled ones as well